lib/editor/components/pattern/PatternStopCard.js

```diff
@@ -117,7 +123,11 @@ const PickupDropoffSelect = (props: PickupDropoffSelectProps) => {
         value={value}
       >
         {pickupDropoffOptions.map(o => (
-          <option key={o.value} value={o.value}>{o.text}</option>
+          <option key={o.value} value={o.value}>
+            {o.value !== '' ? o.text : selectType.includes('continuous')
```

Comment:
`pickupDropoffOptions` is a constant defined in this file (... should it really be, probably not) and I internationalized it in an earlier commit, so those `o.text` values are translated. 
<img width="968" alt="image" src="https://user-images.githubusercontent.com/63798641/217027518-d9608fa5-1ef8-4850-a729-34aaa20ae372.png">

Reply:
This also just reminded me that there's probably a reckoning somewhere down the road with the `translations.txt` file in the spec ....

Reply:
I meant if `selectType` was guaranteed to contain the word `continuous`, but I think that's a gtfs value right? The other fix... might be out of scope here

Reply:
This also made me think this could be a bit brittle, and typing this prop a bit more could help. (https://github.com/ibi-group/datatools-ui/pull/902/commits/17236207fa0366765c3dfd8bc9083f660f50ed8d)